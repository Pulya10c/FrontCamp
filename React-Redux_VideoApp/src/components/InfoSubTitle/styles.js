import styled from 'styled-components';
import { mediaMd, mediaSm } from 'utils/style-utils';

export const InfoSubTitle = styled.h5`
  font-size: 16px;
  color: white;

  ${mediaMd`
    font-size: 14px;
  `};

  ${mediaSm`
    font-size: 12px;
  `};
`;
