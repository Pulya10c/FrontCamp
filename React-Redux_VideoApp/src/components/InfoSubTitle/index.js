import { InfoSubTitle } from './styles';

export default InfoSubTitle;
