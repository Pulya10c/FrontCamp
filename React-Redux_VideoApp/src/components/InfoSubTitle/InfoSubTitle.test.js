import React from 'react';
import { shallow } from 'enzyme';
import InfoSubTitle from './';

describe('Component InfoSubTitle', () => {
  it('should render correctly', () => {
    expect(shallow(<InfoSubTitle />)).toMatchSnapshot();
  });
});
