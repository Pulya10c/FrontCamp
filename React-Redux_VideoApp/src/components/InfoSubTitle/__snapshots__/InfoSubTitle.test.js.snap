// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component InfoSubTitle should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  font-size: 16px;
  color: white;

  ",
          "
  @media (max-width: ",
          "768",
          "px) {
    ",
          "
    font-size: 14px;
  ",
          ";
  }
",
          ";

  ",
          "
  @media (max-width: ",
          "576",
          "px) {
    ",
          "
    font-size: 12px;
  ",
          ";
  }
",
          ";
",
        ],
      },
      "displayName": "styled.h5",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h5",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
